ironfish/src/memPool/memPool.ts

```diff
@@ -89,6 +92,12 @@ export class MemPool {
     return this.transactions.has(hash)
   }
 
+  getFeeRate(mempoolEntry: MempoolEntry): bigint {
+    const rate = mempoolEntry.fee / BigInt(Math.round(mempoolEntry.sizeKb))
+
+    return rate > 0 ? rate : BigInt(1)
+  }
```

Comment:
This should be on the memPool entry to optimize sorting since this will get run multiple times in the compare function.

Reply:
Updated to calculate fee rate when creating the memPool entry. Also, now that we calculate the fee rate using the transaction, I just reused the getFeeRate function from feeEstimator.